snowfakery/output_streams.py

```diff
@@ -125,7 +124,7 @@ def close(self, **kwargs) -> Optional[Sequence[str]]:
 
         Return a list of messages to print out.
         """
-        return super().close()
+        pass
```

Comment:
Discovered that OutputStreams that did not override close() would crash so I fixed that bug.